src/main/java/edu/indiana/dlib/amppd/repository/ItemRepository.java

```diff
@@ -20,7 +20,4 @@
 //	@Query(value = "select i from Item i where lower(i.name) like %:keyword.toLowerCase()% or lower(i.description) like %:keyword.toLowerCase()%")
 //	@Query(value = "select i from Item i where lower(i.name) like %:#{keyword.toLowerCase()}% or lower(i.description) like %:#{keyword.toLowerCase()}%")
 	
-	@Query(value = "select i from Item i where i.name like %:keyword% or i.description like %:keyword%")
-	List<Item> findByKeyword(@Param("keyword") String keyword); 
-
 }

```

Comment:
Is this part removed by intention? I wonder why? findByKeyword is needed for search item

Reply:
Good catch.  Bad merge?  Adding it back in.